Sources/hap-server/main.swift

```diff
@@ -115,7 +115,7 @@ withExtendedLifetime([delegate]) {
         RunLoop.main.run(until: Date(timeIntervalSinceNow: 10))
     } else {
         while keepRunning {
-            RunLoop.current.run(mode: .default, before: Date.distantFuture)
+            RunLoop.current.run()
```

Comment:
This is not equivalent [see: run() RunLoop](https://developer.apple.com/documentation/foundation/runloop/1412430-run):

> If you want the run loop to terminate, you shouldn't use this method. Instead, use one of the other run methods and also check other arbitrary conditions of your own, in a loop. A simple example would be: (...)

Reply:
You are right, I've just reverted this change